CRM/Event/Import/Parser/Participant.php

```diff
@@ -625,4 +625,25 @@ protected function formatValues(&$values, $params) {
     return NULL;
   }
 
+  /**
+   * @deprecated - this is part of the import parser not the API & needs to be moved on out
```

Comment:
This comment makes less sense now the function is moved

Reply:
Yeah - I might loop back to that